sapl/base/views.py

```diff
@@ -915,6 +919,339 @@ def get(self, request, *args, **kwargs):
         return self.render_to_response(context)
 
 
+class ListarInconsistenciasView(PermissionRequiredMixin, ListView):
+    model = get_user_model()
+    template_name = 'base/lista_inconsistencias.html'
+    context_object_name = 'tabela_inconsistencias'
+    permission_required = ('base.list_appconfig',)
+
+    def get_queryset(self):
+        tabela = []
+        tabela.append(
+            ('protocolos_duplicados',
+             'Protocolos duplicados',
+             len(protocolos_duplicados())
+             )
+        )
+        tabela.append(
+            ('protocolos_materias',
+             'Protocolos que excedem o limite de matérias vinculadas',
+             len(protocolos_materias())
+             )
+        )
+        tabela.append(
+            ('materias_protocolo_inexistente',
+             'Matérias Legislativas com protocolo inexistente',
+             len(materias_protocolo_inexistente())
+             )
+        )
+        tabela.append(
+            ('parlamentares_mandatos_intersecao',
+             'Parlamentares com mandatos com interseção',
+             len(parlamentares_mandatos_intersecao())
+             )
+        )
+        tabela.append(
+            ('autores_duplicados',
+             'Autores duplicados',
+             len(autores_duplicados())
+             )
+        )
+        tabela.append(
+            ('bancada_comissao_autor_externo',
+             'Bancadas e Comissões com autor externo',
+             len(bancada_comissao_autor_externo())
+             )
+        )
+        tabela.append(
+            ('legislatura_infindavel',
+             'Legislaturas sem data fim',
+             len(legislatura_infindavel())
+            )
+        )
+
+        return tabela
+
+
+def legislatura_infindavel():
+    legislaturas = []
```

Comment:
Todo o código abaixo pode ser substituído por:

```suggestion
def legislatura_infindavel():
    return [ l for l in Legislatura.objects.filter(data_fim__isnull=True).order_by('-numero')]
```